zaza/charm_lifecycle/utils.py

```diff
@@ -22,6 +23,137 @@
 
 BUNDLE_DIR = "./tests/bundles/"
 DEFAULT_TEST_CONFIG = "./tests/tests.yaml"
+DEFAULT_MODEL_ALIAS = "default_alias"
+
+
+def _model_alias_str_fmt(data):
+    """Convert to model_alias:str format if needed.
+
+    If a string is passed in then the string is mapped to the
+    DEFAULT_MODEL_ALIAS. If a dict is passed then this is a noop.
+
+    :param data: String or Model Alias to data map
+    :type data: str or {}
+    :returns: Model Alias to data map
+    :rtype: {}
```

Comment:
:rtype: Dict[str, ???]

(where ??? might be ANY if it's many different types, or if it is know, what it is)